selenium/firefox/apt/Dockerfile.tmpl

```diff
@@ -6,5 +6,18 @@ ARG CLEANUP
 RUN  \
         ( [ "$CLEANUP" != "true" ] && rm -f /etc/apt/apt.conf.d/docker-clean ) || true && \
         apt-get update && \
-        apt-get -y --no-install-recommends install firefox=$VERSION && \
+        apt-get -y --no-install-recommends install pulseaudio firefox=$VERSION && \
         ($CLEANUP && rm -Rf /tmp/* && rm -Rf /var/lib/apt/lists/*) || true
+
+RUN useradd seluser \
```

Comment:
I don't get - what is the purpose of running this image from non-root user? Without pulseaudio it works like a charm under root. Also we are using `selenium` as non-root user name, not `seluser`. Here is a code snippet: https://github.com/aerokube/selenoid-images/blob/master/selenium/chrome/Dockerfile.driver.tmpl#L7-L11

Reply:
Unfortunately `pulseaudio` needs to be executed with a non-root user. I have committed a new patch renaming this user to `selenium` as requested.